data/mods/Magiclysm/materials.json

```diff
@@ -89,6 +89,28 @@
     "burn_products": [ [ "scrap_bronze", 1 ] ],
     "compacts_into": [ "orichalcum_lump", "orichalcum_sliver" ]
   },
+  {
+    "type": "material",
+    "id": "mithril_metal",
+    "name": "Mithril",
+    "density": 20,
+    "bash_resist": 13,
+    "specific_heat_liquid": 0.82,
+    "specific_heat_solid": 0.45,
+    "latent_heat": 273,
+    "cut_resist": 20,
+    "bullet_resist": 16,
+    "acid_resist": 10,
+    "fire_resist": 10,
+    "elec_resist": 10,
+    "chip_resist": 40,
+    "repaired_with": "mithril_ingot",
+    "dmg_adj": [ "marked", "dented", "smashed", "shattered" ],
+    "bash_dmg_verb": "dented",
+    "cut_dmg_verb": "scratched",
+    "burn_products": [ [ "scrap_bronze", 1 ] ],
```

Comment:
this is wrong, it should be scrap aluminum or whatever the id is.

Reply:
there is no burn_products for aluminium should i switch to "scrap" ?

Reply:
no, because `scrap` is steel.